content/docs/rfcs/45/README.md

```diff
@@ -0,0 +1,338 @@
+tags:
+editor: Daniel Kaiser <danielkaiser@status.im>
+contributors:
+---
+
+# Abstract
+
+This document lists adversarial models and attack-based threats relevant in the context of Waku v2.
+
+# Motivation and Background
+
+Future versions of this document will serve as a comprehensive list of adversarial models and attack based threats relevant for [Waku v2](/spec/10/).
+The main purpose of this document is being a linkable resource for specifications that address protection as well as mitigation mechanisms within the listed models.
+
+Discussing and introducing countermeasures to specific attacks in specific models is out of scope for this document.
+Analyses and further information about Waku's properties within these models may be found in our *Waku v2 Anonymity Analysis* series of research log posts:
+
+* [Part I: Definitions and Waku Relay](https://vac.dev/wakuv2-relay-anon)
+
+Note: This document adds to the adversarial models and threat list discussed in our [research log post](https://vac.dev/wakuv2-relay-anon).
+It does not cover analysis of Waku, as the research log post does.
+Future versions of this document will extended the adversarial models and threat list.
+
+# Informal Definitions: Security, Privacy, and Anonymity
+
+The concepts of security, privacy, and anonymity are linked and have quite a bit of overlap.
+
+## Security
+
+Of the three, [Security](https://en.wikipedia.org/wiki/Information_security) has the clearest agreed upon definition,
+at least regarding its key concepts: *confidentiality*, *integrity*, and *availability*.
+
+* confidentiality:  data is not disclosed to unauthorized entities.
+* integrity: data is not modified by unauthorized entities.
+* availability: data is available, i.e. accessible by authorized entities.
+
+While these are the key concepts, the definition of information security has been extended over time including further concepts,
+e.g. [authentication](https://en.wikipedia.org/wiki/Authentication) and [non-repudiation](https://en.wikipedia.org/wiki/Non-repudiation).
+
+## Privacy
+
+Privacy allows users to choose which data and information
+
+* they want to share
+* and with whom they want to share it.
+
+This includes data and information that is associated with and/or generated by users.
+Protected data also comprises metadata that might be generated without users being aware of it.
+This means, no further information about the sender or the message is leaked.
+Metadata that is protected as part of the privacy-preserving property does not cover protecting the identities of sender and receiver.
+Identities are protected by the [anonymity property](#anonymity).
+
+Often privacy is realized by the confidentiality property of security.
+This neither makes privacy and security the same, nor the one a sub category of the other.
+While security is abstract itself (its properties can be realized in various ways), privacy lives on a more abstract level using security properties.
```

Comment:
Perhaps I'm missing the point here - are we saying security is abstract, but privacy is _more_ abstract?